src/ansys/dpf/core/operators/result/__init__.py

```diff
@@ -1,7 +1,6 @@
 from .acceleration import acceleration
 from .acceleration_X import acceleration_X
 from .acceleration_Y import acceleration_Y
-from .acceleration_Z import acceleration_Z
```

Comment:
For some reason the update erases the acceleration_Z operator file

Reply:
For some reason the `acceleration_z.py` file is considered as renamed to `magnetic_vector_potential.py`!